staging/src/k8s.io/kubectl/pkg/explain/v2/funcs.go

```diff
@@ -236,5 +236,19 @@ func WithBuiltinTemplateFuncs(tmpl *template.Template) *template.Template {
 			}
 			return cur
 		},
+		// styleVerboseRecursive creates an string for styling the verboseRecursive based
+		// on if the line is a filed or description/empty line.
+		"styleVerboseRecursive": func(level int, field bool) string {
```

Comment:
I'm curious what Alex thinks, but maybe we can embed that in the `indent` function? along with the flag so that we don't have to pass that to the schema? Curious about your opinions.

Reply:
Or provide a way to indent in the template in a more flexible way?

Reply:
The reason i've created a func here was that we are going to use some brand new styling and maybe we don't need to change the `indent` or template so much and use it like this. But if you or Alex thinks merging it to template or `indent` function would be better approach i'm happy to change it.